### ISSUE TITLE
@stryker-mutator/core contains vulnerability

### ISSUE THREAD

**Issue by peternedap** [stryker-mutator/stryker#2278]
**Is your maintenance request related to a problem? Please describe.**
`@stryker-mutator-core` doesn't pass `yarn audit` due to a high risk vulnerability (Command Injection).

**Describe the solution you'd like**
An update to `tree-kill` would fix this.

**Describe alternatives you've considered**
Could this be temporarily fixed with an override (via peer dependency?)

**Additional context**
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Command Injection                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ tree-kill                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=1.2.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @stryker-mutator/core                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @stryker-mutator/core > tree-kill                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1432                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by kmdrGroch** [comment]
Thanks for opening this issue!
I hope I could fix it with #2279 :) You might want to take a look (I don't have yarn so I don't know if it still marks it as a vulnerability)
Cheers!

**Comment by peternedap** [comment]
Your fix seems to resolve the vulnerability :)

**Comment by kmdrGroch** [comment]
this should be fixed as was merged in #2295

**Comment by kmdrGroch** [comment]
@peternedap stryker 4 has it fixed. Could it be closed? :)
